odoo/addons/test_http/tests/test_session.py

```diff
@@ -242,6 +242,47 @@ def test_session10_explicit_session(self):
         )
         self.assertEqual(admin_session.debug, '1')
 
+    def test_session11_items_accessibility(self):
+        session = self.authenticate('admin', 'admin')
+        # Access default items
+        # These items are always accessible with properties
+        # even if it no longer exists in the data
+        login_value = 'other admin'
+        self.assertTrue('login' in session)
```

Comment:
otherwise the assertion message is "false is not true"
```suggestion
        self.assertIn('login', session)
```